### PR TITLE
Add documentation for renault.ac_set_schedules

### PR DESCRIPTION
**Pull request by vhkristof**
## Proposed change

This PR adds documentation for the new service that is implemented by https://github.com/home-assistant/core/pull/125006

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125006
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new action to update air conditioning schedules for vehicles, allowing users to set multiple schedules with enhanced flexibility.
	- Documentation now includes examples to aid user understanding of the new scheduling functionality.

### CODE REVIEW

File: source/_integrations/renault.markdown [hunk 1]
```diff
@@ -58,6 +58,39 @@ Cancel A/C on vehicle.
...
+Update AC schedule on vehicle.
+
+  | Data attribute | Required | Description | Example |
+  | ---------------------- | -------- | ----------- | ------- |
+  | `vehicle`| yes | device_id of the vehicle | |
+  | `schedules` | yes | Schedule details. Can be a single schedule or a list of schedules | see [example below](#ac_schedule_example) |
+
+Notes:
+
+- `schedules` can be in the form `{'id':1,...}` when updating a single schedules, or in the form `[{'id':1,...},{'id':2,...},...]` when updating multiple schedules within the same call
```
Comment (c0ffeeca7) [inline source/_integrations/renault.markdown]:
```suggestion
- `schedules` can be in the form `{'id':1,...}` when updating a single schedule, or in the form `[{'id':1,...},{'id':2,...},...]` when updating multiple schedules within the same call.
```

File: source/_integrations/renault.markdown [hunk 2]
```diff
@@ -58,6 +58,39 @@ Cancel A/C on vehicle.
...
+- the `id` is compulsory on each `schedule` (should be 1 to 5 depending on the vehicle)
```
Comment (c0ffeeca7) [inline source/_integrations/renault.markdown]:
```suggestion
- The `id` is compulsory on each `schedule` (should be 1 to 5 depending on the vehicle).
```

File: source/_integrations/renault.markdown [hunk 3]
```diff
@@ -58,6 +58,39 @@ Cancel A/C on vehicle.
...
+- the `activated` flag is an optional boolean. If it is not provided, then the existing flag will be kept as is.
```
Comment (c0ffeeca7) [inline source/_integrations/renault.markdown]:
```suggestion
- The `activated` flag is an optional boolean. If it is not provided, then the existing flag will be kept as is.
```

File: source/_integrations/renault.markdown [hunk 4]
```diff
@@ -58,6 +58,39 @@ Cancel A/C on vehicle.
...
+- the `monday` to `sunday` elements are optional. If they are not provided, then the existing settings will be kept for each day. If they are provided as None, then the existing setting will be cleared. If a value is provided, it must conform to this format `{'readyAtTime':'T12:00Z'}` where start time is in UTC format and the duration is in minutes.
```
Comment (c0ffeeca7) [inline source/_integrations/renault.markdown]:
```suggestion
- The `monday` to `sunday` elements are optional. If they are not provided, then the existing settings will be kept for each day. If they are provided as None, then the existing setting will be cleared. If a value is provided, it must conform to this format `{'readyAtTime':'T12:00Z'}` where start time is in UTC format and the duration is in minutes.
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @vhkristof :+1:

**Comment by vhkristof** [comment]
Will create a new PR. Messed up with wrong merge :-/

**Comment by vhkristof** [comment]
See #34636

**Pull request closed without merging**